mlrun/config.py

```diff
@@ -398,6 +398,7 @@
             "project_secret_name": "mlrun-project-secrets-{project}",
             "auth_secret_name": "mlrun-auth-secrets.{hashed_access_key}",
             "env_variable_prefix": "MLRUN_K8S_SECRET__",
+            "global_secret_name": None,
```

Comment:
I think this is too generic name, I would add something that states that its related to runtimes

Reply:
It's inside a specific section of the config - it's full name is `secret_stores.kubernetes.global_secret_name` (as written in the description)

Reply:
I mean that `global_secret_name` can be used for multiple purposes, for now, we are using that to enrich users functions, next time, user might want to pass other configuration in a secret so either we will need to provide another param configuration or have sub-sections inside the secret that we expect.
e.g for what we are trying to achieve now, we would have a root key which will be called `functions` and inside of it will be all the user auto enrichments for all functions, that way user won't need multiple secret configuration for us and it will be easier to expend.